aiomonitor/monitor.py

```diff
@@ -388,13 +404,18 @@ def do_exit(self) -> None:
 
     def do_console(self) -> None:
         """Switch to async Python REPL"""
-        if not self._console_enabled:
+        if not self._monitor._console_enabled:
             self._sout.write('Python console disabled for this sessiong\n')
             self._sout.flush()
 
-        if self._console_future is not None:
-            self._console_future.result()
-        console_proxy(self._sin, self._sout, self._host, self._console_port)
+        if self._monitor._console_future is not None:
+            self._monitor._console_future.result()
```

Comment:
I am wondering, if we can wait for console server to start when we init it instead of here. This should simplify things a bit.

Reply:
I'm not sure I understand what you mean. Could you elaborate?

Reply:
I just thinking if it is better to move `_console_future.result()` closer where we actually create this future object.
https://github.com/aio-libs/aiomonitor/blob/230a351152dd45ec6b5f98ae56d76678653c4b19/aiomonitor/monitor.py#L108-L109

In current implementation we create future and wait it once we need to be sure that python console is started, instead doing this should we just wait console immediately?

Reply:
reworked this part of code in master